firebase-functions/functions/src/operations/RoundOperations.ts

```diff
@@ -35,3 +46,24 @@ function getResult(player: QueryDocumentSnapshot<DocumentData>, allPlayers: Quer
       ? Result.Lost
       : Result.Won;
 }
+
+async function getRoom(roomId: string): Promise<Room> {
+  const roomDoc = await admin.firestore().collection(collections.rooms).doc(roomId).get();
+  return roomDoc.data() as Room;
+}
+
+function defineLastOneActive(playerChoices: PlayerChoice[], typeOfGame?: GameType): string | undefined {
+  return playerChoices.find(player => player.result === Result.Draw)
+    ? undefined
+    : typeOfGame === GameType.Loser
+      ? playerChoices.find(player => player.result === Result.Lost)?.playerId
+      : playerChoices.find(player => player.result === Result.Won)?.playerId;
+}
+
+function createNewGame(round: Round, typeOfGame?: GameType): Game {
+  const lastOneActive = defineLastOneActive(round.playerChoices, typeOfGame);
+  return {
+    rounds: [round],
+    lastOneActive: lastOneActive
+  } as Game
```

Comment:
There is some linting rules violated in this file (double quotes and missing semicolons). It seems to me that on CI build the linter is only run for the angular project.
I suggest you configure mighty rider to detect them until we fix the build.